event_operating_unit/views/event_track_views.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record model="ir.ui.view" id="view_event_track_tree_inherit_ou">
+        <field name="name">event.track.tree</field>
+        <field name="model">event.track</field>
+        <field name="inherit_id" ref="website_event_track.view_event_track_tree" />            
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name"/>
+                <field name="active" invisible="1"/>
+                <field name="operating_unit_id" groups="operating_unit.group_multi_operating_unit" />
+            </tree>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="view_event_track_search">
+        <field name="name">event.track.search</field>
+        <field name="model">event.track</field>
+        <field name="inherit_id" ref="website_event_track.view_event_track_search" />
+        <field name="arch" type="xml">
+            <search>
+                <field name="operating_unit_id" string="Operating Unit" groups="operating_unit.group_multi_operating_unit" />
+            </search>
+            <xpath expr="//group[1]">
```

Comment:
A better XPath would be `//filter[@context="{'group_by':'event_id'}"]` with `postion="after"`